string topic
---
sensor_msgs/CompressedImage result
string json_string
